steps/upload-to-sauce-labs/0.0.1/step.yml

```diff
@@ -0,0 +1,98 @@
+#
+# A couple of useful guides & docs:
+#
+# - Main Bitrise CLI docs: https://github.com/bitrise-io/bitrise/tree/master/_docs
+# - Step Development Guideline: https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md
+# - Bitrise.yml format spec: https://github.com/bitrise-io/bitrise/blob/master/_docs/bitrise-yml-format-spec.md
+# - Bitrise docs: http://devcenter.bitrise.io/
+# - Bitrise CLI guides: http://devcenter.bitrise.io/bitrise-cli/
+
+title: |-
+  Upload to Sauce Labs
+summary: |
+  Upload an APK/AAB/IPA/ZIP artifact to Sauce Labs.
+description: |
+  Upload your build file to Sauce Labs for remote device testing.
+  Requires an APK, AAB, IPA, or ZIP file within the workflow, Sauce Labs username, access key, and file name.
+  You can optionally provide a build path and request url.
+
+  Required fields are:
+  - sauce_username
+  - sauce_access_key
+  - sauce_app_name
+
+  Optional fields are:
+  - artifact_path
+  - data_center_url
+website: https://github.com/camseda/bitrise-step-upload-to-sauce-labs
+source_code_url: https://github.com/camseda/bitrise-step-upload-to-sauce-labs
+support_url: https://github.com/camseda/bitrise-step-upload-to-sauce-labs/issues
+published_at: 2024-03-26T12:38:22.13670071+02:00
+source:
+  git: https://github.com/camseda/bitrise-step-upload-to-sauce-labs.git
+  commit: 291be42de92c3c1fe0ed1823969690640d19ae37
+project_type_tags:
+  - ios
+  - android
+
+type_tags:
+  - deploy
+
+is_always_run: false
+is_skippable: false
+run_if: ""
+
+deps:
+  brew:
+    - name: git
```

Comment:
Is the `git` cli tool a dependency of this step? By looking at the source code `git` command isn't invoked directly.